### PR TITLE
Translate to Traditional Chinese

### PR DESCRIPTION
**Pull request by MilesChou**

### CODE REVIEW

File: content/5-min-overview/index.md [hunk 1]
```diff
@@ -35,7 +55,19 @@ called 'trunk' at all.
 
 There are many deciding factors before a development team settles on Trunk-Based Development, but here is a short overview 
 of the practices if they do:
+-->
+
+## 它是什麼？
+
+{{< note title="Notes" >}}
 
+* 在本網站上，提到〝開發者〞一詞時，指的是對相同可建置項目進行自動化測試的〝測試人員〞。
```
Comment (minchao) [inline content/5-min-overview/index.md:64]: To clarify, I suggest revising it to: 

```
* 在本網站上，提到〝開發者〞一詞時，也適用於對相同可建置項目進行自動化測試的〝測試人員〞。
```

Comment (MilesChou) [reply content/5-min-overview/index.md:64]: Thank you for your reply. We will discuss this internally. If your suggestion is better, would you authorize us to use it directly?

Comment (minchao) [reply content/5-min-overview/index.md:64]: Absolutely, feel free to implement the suggestion.

**Comment by paul-hammant** [comment]
Hey team, you're able to Run Hugo locally, to see your work in progress?

**Comment by tedmax100** [comment]
> Hey team, you're able to Run Hugo locally, to see your work in progress?

Hi Paul,

I am one of the translation volunteer team. 

The screenshot is of the interface when I launched using Hugo, and it looks just like the English website. We will also use Hugo to launch and check the content when we merge. 

Thank you for the reminder.

image

**Comment by MilesChou** [comment]
@paul-hammant Perhaps we can merge some of the translations to get the website up and running first. Then, once the team completes the remaining parts, we can submit another PR.

**Comment by MilesChou** [comment]
We have completed all the translation work and are ready to merge.

**Comment by paul-hammant** [comment]
yay!

**Pull request merged**